### DISCUSSION TITLE
멀티플레이 예제에 있는 에셋들을 상업적으로 사용해도 괜찮나요?

### DISCUSSION THREAD

**Discussion by icjoon** [naverz/zepeto-studio-kor#560] [5. Q&A]
안녕하세요 제페토 월드개발을 공부하는 중인데 빌드잇 패키지에만 있는 에셋들로 구성하기에 한계가 느껴지다가
멀티플레이 예제에 있는 에셋들도 같이 사용하면 좋을거라 생각하는데 어쩌다 제가 라이센스 위반을 하는 실수를 방지하기 위해서 이렇게 질문을 남깁니다.

**Comment by cheolhui-lee** [comment]
안녕하세요.
제페토 월드 내에서 사용하시는 것이라면 문제없이 사용가능합니다.
고맙습니다.
